google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -130,6 +130,7 @@ clouddebug.stop.and.create.new.session=A cloud debugger session is already runni
 clouddebug.stop.background.and.create.new.session=A cloud debugger session is listening in the background.\nWould you like to stop it and start a new session?
 clouddebug.logout.additional.message=Any Cloud Debugger sessions listening in the background will be stopped.
 clouddebug.background.listener.error.title=Error while connecting to Cloud Debugger backend
-clouddebug.background.listener.error.message=A cloud debugger process is listening in the background but cannot access the backend. Verify that there is a Google account signed in with access to the project.
+clouddebug.background.listener.access.error.message=Access denied while listening for Cloud Debugger snapshots for project <strong>{0}</strong>. Verify that you are signed in with a Google account that has the appropriate permissions set.
```

Comment:
just a question? Is it meaningful to include the type (html) of the message when not standard text